daemon/daemon.go

```diff
@@ -603,7 +603,7 @@ func NewDaemon(config *Config, registryService registry.Service, containerdRemot
 	}
 
 	logrus.Debugf("Max Concurrent Downloads: %d", *config.MaxConcurrentDownloads)
-	d.downloadManager = xfer.NewLayerDownloadManager(d.layerStore, *config.MaxConcurrentDownloads)
+	d.downloadManager = xfer.NewLayerDownloadManager(d.layerStore, *config.MaxConcurrentDownloads, filepath.Join(config.Root, "incomplete_downloads"))
```

Comment:
Make this shorter. Something like `incomplete` is fine.

Path bloat can cause issues in practice.